src/main/java/com/mraof/minestuck/block/MSBlocks.java

```diff
@@ -424,7 +445,14 @@ public static void registerBlocks(final RegistryEvent.Register<Block> event)
 		registry.register(new DecorBlock(Block.Properties.create(Material.IRON).hardnessAndResistance(0.5F).sound(SoundType.METAL), MSBlockShapes.BLENDER).setRegistryName("blender"));
 		registry.register(new DecorBlock(Block.Properties.create(Material.ROCK).hardnessAndResistance(0.5F), MSBlockShapes.CHESSBOARD).setRegistryName("chessboard"));
 		registry.register(new DecorBlock(Block.Properties.create(Material.ROCK).hardnessAndResistance(0.5F), MSBlockShapes.FROG_STATUE).setRegistryName("mini_frog_statue"));
+		registry.register(new DecorBlock(Block.Properties.create(Material.ROCK).hardnessAndResistance(1.0F).sound(SoundType.STONE), MSBlockShapes.NAKAGATOR_STATUE).setRegistryName("nakagator_statue"));
+		registry.register(new DecorBlock(Block.Properties.create(Material.IRON).hardnessAndResistance(-1.0F, 25.0F).sound(SoundType.METAL), MSBlockShapes.CALEDFWLCH_PEDESTAL_EMPTY).setRegistryName("caledfwlch_pedestal_empty"));
+		registry.register(new DecorBlock(Block.Properties.create(Material.IRON).hardnessAndResistance(5.0F).sound(SoundType.METAL), MSBlockShapes.LOOT_CHEST_EMPTY).setRegistryName("loot_chest_empty"));
 		registry.register(new GlowystoneWireBlock(Block.Properties.create(Material.MISCELLANEOUS).hardnessAndResistance(0.0F).lightValue(16).doesNotBlockMovement()).setRegistryName("glowystone_dust"));
+
+		registry.register(new LootBlock(Block.Properties.create(Material.IRON).hardnessAndResistance(5.0F).sound(SoundType.METAL), MSBlockShapes.LOOT_CHEST).setRegistryName("loot_chest"));
+		registry.register(new LootBlock(Block.Properties.create(Material.IRON).hardnessAndResistance(5.0F).sound(SoundType.METAL), MSBlockShapes.WOODEN_LOOT_CHEST).setRegistryName("wooden_loot_chest"));
+		registry.register(new LootBlock(Block.Properties.create(Material.IRON).hardnessAndResistance(-1.0F, 25.0F).sound(SoundType.METAL), MSBlockShapes.CALEDFWLCH_PEDESTAL_FULL).setRegistryName("caledfwlch_pedestal_full"));
```

Comment:
Why -1 hardness?

Reply:
To avoid mining, movement, or destruction of the block to reflect the pedestals unyielding nature in the comic... ignoring the use of a pillar to break the sword itself

Reply:
> unyielding nature in the comic
Are you sure that's a thing that we would want to go by in the mod? Is it even a thing in comic? There are probably some situations where unbreakable blocks would be good, but I don't agree that this is one of those.